dcc-submission-validator/src/test/java/org/icgc/dcc/submission/validation/accession/ega/FileOperationHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.icgc.dcc.submission.validation.accession.ega;
+
+import java.io.*;
+
+/**
+ * Copyright (c) 2017 The Ontario Institute for Cancer Research. All rights reserved.
+ * <p>
+ * This program and the accompanying materials are made available under the terms of the GNU Public License v3.0.
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ * <p>
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+class FileOperationHelper {
+
+  public static void copyFileFromClasspathToTmpDir(String classpath, String fileName, String targetPath){
+    try {
+
+      File target = new File(targetPath + "/" + fileName);
+      target.createNewFile();
+
+      FileOutputStream fos = new FileOutputStream(target);
+      InputStream is = ShellScriptDownloaderTest.class.getResourceAsStream(classpath + "/" + fileName);
+      byte[] buffer = new byte[512];
+      int len = 0;
+      while((len = is.read(buffer)) > 0){
+        fos.write(buffer, 0, len);
+      }
+      fos.flush();
+      fos.close();
```

Comment:
because you are also catching the `IOException` from the file creation and cant easily use a try-with-resources your closing of the stream should be in the `finally` block.